diff --git a/node_modules/@nx/rollup/src/executors/rollup/lib/update-package-json.js b/node_modules/@nx/rollup/src/executors/rollup/lib/update-package-json.js
index cc9acc7..6e0f938 100644
--- a/node_modules/@nx/rollup/src/executors/rollup/lib/update-package-json.js
+++ b/node_modules/@nx/rollup/src/executors/rollup/lib/update-package-json.js
@@ -16,7 +16,7 @@ function updatePackageJson(options, context, target, dependencies, packageJson)
         packageJson.exports['./package.json'] = './package.json';
     }
     if (hasEsmFormat) {
-        const esmExports = getExports(Object.assign(Object.assign({}, options), { fileExt: '.esm.js' }));
+        const esmExports = getExports(Object.assign(Object.assign({}, options), { fileExt: '.mjs' }));
         packageJson.module = esmExports['.'];
         if (!hasCjsFormat) {
             packageJson.type = 'module';
@@ -31,7 +31,7 @@ function updatePackageJson(options, context, target, dependencies, packageJson)
         }
     }
     if (hasCjsFormat) {
-        const cjsExports = getExports(Object.assign(Object.assign({}, options), { fileExt: '.cjs.js' }));
+        const cjsExports = getExports(Object.assign(Object.assign({}, options), { fileExt: '.cjs' }));
         packageJson.main = cjsExports['.'];
         if (!hasEsmFormat) {
             packageJson.type = 'commonjs';
